[2025-05-05T02:10:03.642+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-05-05T02:09:00+00:00 [queued]>
[2025-05-05T02:10:03.650+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-05-05T02:09:00+00:00 [queued]>
[2025-05-05T02:10:03.651+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-05T02:10:03.666+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): load_mysql_data> on 2025-05-05 02:09:00+00:00
[2025-05-05T02:10:03.672+0000] {standard_task_runner.py:60} INFO - Started process 11840 to run task
[2025-05-05T02:10:03.675+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mymind_mongo_to_mysql_etl', 'load_mysql_data', 'scheduled__2025-05-05T02:09:00+00:00', '--job-id', '28484', '--raw', '--subdir', 'DAGS_FOLDER/mymind_etl_dag.py', '--cfg-path', '/tmp/tmpunzrsdqk']
[2025-05-05T02:10:03.677+0000] {standard_task_runner.py:88} INFO - Job 28484: Subtask load_mysql_data
[2025-05-05T02:10:03.737+0000] {task_command.py:423} INFO - Running <TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-05-05T02:09:00+00:00 [running]> on host 9548ec498b56
[2025-05-05T02:10:03.855+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mymind_mongo_to_mysql_etl' AIRFLOW_CTX_TASK_ID='load_mysql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-05T02:09:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-05T02:09:00+00:00'
[2025-05-05T02:10:03.857+0000] {mymind_etl_dag.py:285} INFO - Iniciando carga a MySQL: 4 usuarios, 18 transcripciones.
[2025-05-05T02:10:03.868+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-05-05T02:10:03.879+0000] {mymind_etl_dag.py:304} INFO - Cargando 4 usuarios en la tabla users...
[2025-05-05T02:10:03.881+0000] {mymind_etl_dag.py:329} INFO - Usuarios cargados exitosamente.
[2025-05-05T02:10:03.881+0000] {mymind_etl_dag.py:342} INFO - Cargando 18 transcripciones en la tabla transcriptions...
[2025-05-05T02:10:03.884+0000] {mymind_etl_dag.py:372} INFO - Transcripciones cargadas exitosamente.
[2025-05-05T02:10:03.885+0000] {mymind_etl_dag.py:379} INFO - Carga a MySQL completada con Ã©xito.
[2025-05-05T02:10:03.885+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-05T02:10:03.896+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mymind_mongo_to_mysql_etl, task_id=load_mysql_data, execution_date=20250505T020900, start_date=20250505T021003, end_date=20250505T021003
[2025-05-05T02:10:03.927+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-05T02:10:03.955+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
